### ISSUE TITLE
Zen Archer Flurry of Blows (Arrows) does not trigger an extra bow attack.

### ISSUE THREAD

**Issue by Krayte** [SolastaMods/SolastaUnfinishedBusiness#4408]
As the title says either Flurry of Blows doesn't change to Flurry of Arrows or selecting Flurry of Blows isn't handing the ability over to bows.

![2024-03-09_21-03-04](https://github.com/SolastaMods/SolastaUnfinishedBusiness/assets/42162713/c9192349-84bb-4ddb-8936-aeb525bb18cf)

![2024-03-09_21-04-17](https://github.com/SolastaMods/SolastaUnfinishedBusiness/assets/42162713/6f2aad78-0b57-4e1a-98b5-b612904c857f)

![2024-03-09_21-04-58](https://github.com/SolastaMods/SolastaUnfinishedBusiness/assets/42162713/d1bdc320-43d7-4a6d-9871-bbb3cb1c8373)

[AutoSave_1.zip](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/14548920/AutoSave_1.zip)

**Comment by ThyWoof** [comment]
Do you have the setting that automatically enables flurry of blows under mod UI checked?

**Comment by Krayte** [comment]
I don't think so, I have to manually engage it.  I couldn't find that option exactly but here is the section in my UB1 that refers to Monk abilities.  If that option is somewhere else, let me know.

![2024-03-09_23-17-46](https://github.com/SolastaMods/SolastaUnfinishedBusiness/assets/42162713/3d6fc4a7-5e08-4d32-9ba7-4163363f585e)

**Comment by ThyWoof** [comment]
cannot repro on my end with a brand new hero.
![image](https://github.com/SolastaMods/SolastaUnfinishedBusiness/assets/13087646/e11aa715-d1cd-48ce-aec5-3e74979e1c0e)

I suggest a RESPEC on your end.

**Comment by Krayte** [comment]
I tested a few things and found this.  The character I am playing is a Cleric / Zen Monk.  

If I build like so it reproduces (I even respec'd twice to make sure):
Cleric 1Monk 2Monk 3Monk 4 (Zen Archer)Flurry of Arrows does not trigger
If I build:Monk 1Cleric 2Monk 3Monk 4Flurry of Arrows triggers and works
My guess is this is only broken when you choose a different class at 1st level?  Or maybe it's only specific to when you choose Cleric First?
Either way I fixed it my save but wanted to let you know in case you wanted to dig deeper.
Thanks for all your help!
-Krayte
    On Sunday, March 10, 2024 at 08:34:33 AM PDT, Paulo Monteiro ***@***.***> wrote:  
 
 


cannot repro on my end with a brand new hero.
image.png (view on web)

I suggest a RESPEC on your end.
